src/ydata_synthetic/postprocessing/regular/inverse_preprocesser.py

```diff
@@ -1,45 +1,46 @@
 # Inverts all preprocessing pipelines provided in the preprocessing examples
```

Comment:
Changes in this script consist in extending the inverse support to the MinMaxScaler